Source/WebCore/loader/appcache/ApplicationCacheHost.cpp

```diff
@@ -280,7 +280,7 @@ bool ApplicationCacheHost::maybeLoadSynchronously(ResourceRequest& request, Reso
 
 void ApplicationCacheHost::maybeLoadFallbackSynchronously(const ResourceRequest& request, ResourceError& error, ResourceResponse& response, RefPtr<SharedBuffer>& data)
 {
-    // If normal loading results in a redirect to a resource with another origin (indicative of a captive portal), or a 4xx or 5xx status code or equivalent,
+    // If normal loading results in a redirect to a resource with another origin (indicative of a Lockdown), or a 4xx or 5xx status code or equivalent,
```

Comment:
Probably want to undo this one?

Reply:
Whoops, good catch! Looks this one is really about captive portals 😓.